js/demo-app/src/index.js

```diff
@@ -92,6 +92,10 @@ document.addEventListener("DOMContentLoaded", () => {
         // On navRequest & routing mode, change url
       } else {
         setRoute(data.detail.fragment);
+        let activeNavItem = NAV_CONFIGS.find(toMatch => {
```

Comment:
I think it would work if you pass `data.detail.fragment` into updateActiveNav and just remove the check on `urlRoutingEnabled` in that method.

I haven't tested it, but that was my intention when I added that string param.  Sorry that i named it `fqRouteName`.  That wasn't the best